docs/content/usage-reporting.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Usage Reporting
+description: |
+How to enable usage reporting for NGINX Ingress Controller and how to view the usage data through the API.weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: ["???"]
+---
+
+
+This document outlines how to enable usage reporting for NGINX Ingress Controller and how to view the usage data through the API.
+
+## Overview
+
+The Flexible Consumption Program (FCP) is a new pricing offer for NGINX Ingress Controller. The  pricing model is based on the number of NGINX Ingress Controller instances in a cluster. The number is reported to NGINX Management Suite, which is used to calculate the license cost.
+
+NGINX Cluster Connector is a Kubernetes controller that connects to the NGINX Management Suite and reports the number of NGINX Ingress Controller nodes in the cluster. The NGINX Cluster Connector is deployed as a Kubernetes Deployment in the same cluster where NGINX Ingress Controller is deployed.
+
+To use the NGINX Cluster Connector, you must have access to NGINX Management Suite. For more information, see [NGINX Management Suite](https://www.nginx.com/products/nginx-management-suite/).
```

Comment:
```suggestion
Usage Reporting is a Kubernetes controller that connects to the NGINX Management Suite and reports the number of NGINX Ingress Controller nodes in the cluster. Usage Reporting is installed as a Kubernetes Deployment in the same cluster as the NGINX Ingress Controller whose nodes you would like reported.

To use Usage Reporting, you must have access to NGINX Management Suite. For more information, see [NGINX Management Suite](https://www.nginx.com/products/nginx-management-suite/).  Usage Reporting is part of the new Flexible Consumption Program for NGINX Ingress Controller, calculating costs.
```
I'm inclined to link to the F5 page about the Flexible Consumption Program, but it's unclear to me if that's the umbrella process which people engage with.